app/services/base_suggestions_service.rb

```diff
@@ -35,4 +36,8 @@ def handle_commodity_record(_commodity)
   def handle_search_reference_record(_search_reference)
     nil
   end
+
+  def hidden_goods_nomenclature_item_ids
```

Comment:
what about matching the table name?
`def hidden_goods_nomenclature_ids`

also, it is shorter :)